index.html

```diff
@@ -63,6 +59,169 @@ <h2>I'am a Software Developer</h2>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        <div class="project project4">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+       
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+
+        <div class="project project5">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+    
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+
+        <div class="project project6">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        <div class="project project7">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        </section>
+               <!--ABOUT SECTION-->
+<section>
+ <div class="aboutsection">        
+<div class="about-part">
+<h2 class="aboutmehead">About me</h2>
+<p class="aboutpara">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<button type="button" class="buttn">Get My Resume</button>
```

Comment:
- [ ] Kindly align the content of the about me section to the left to look similar to the Figma template 👇🏽 

| **Figma template** | **Your template** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192119602-4915dba1-508e-4412-b2a6-6826069435d4.png) | ![image](https://user-images.githubusercontent.com/58642949/192119608-2d2f2240-8589-4ba8-bbba-eb0191d65142.png) |

Reply:
Good job making the changes as requested by the previous reviewer.